python/lsst/sims/alertsim/avro_utils.py

```diff
@@ -9,13 +9,14 @@
 
 from timeit import default_timer as timer
 import json
+from collections import defaultdict
 
 def _raise_no_avro(method_name):
     msg = "To use %s you must install avro from https://avro.apache.org" % method_name
     raise RuntimeError(msg)
 
 
-def catsim_to_avro(list_of_query_dicts, schemaURI='avsc/diasource.avsc'):
+def catsim_to_avro(list_of_alert_dicts, session_dir, schemaURI='avsc/diasource.avsc'):
```

Comment:
You need to update the docstring for this method to mention `session_dir`

Reply:
ok